examples/text_to_image/train_text_to_image.py

```diff
@@ -69,10 +69,10 @@ def save_model_card(
     args,
     repo_id: str,
     images: list = None,
-    repo_folder: str = None,
+    repo_folder: str = ".",
 ):
     img_str = ""
-    if len(images) > 0:
+    if images is not None:
```

Comment:
Even better could be `if images`. But we try to be more explicit throughout the libraries in the HF ecosystem. So, would prefer sticking to the original check here.

Reply:
If `images` somehow takes its default value, doesn't `len(images)` throw an error?